### PR TITLE
Endpoint_to_delete_CVs_using parameter_ID

### PR DESCRIPTION
**Pull request by CharlesKimMaina**
# Title: Endpoint_to_delete_CVs_using parameter_ID. 

Fixes https://github.com/HackYourFuture-CPH/rate-my-cv/issues/3

# How to test? You can test it on POSTMAN,.,. I have gotten below results; 1 CV was deleted as it has a certain ID

CV_deleted

Another CV wasn't deleted as it's ID don't exist.
CV_2b_deleted, dont exists

# Checklist

- [*] I have performed a self-review of my own code
- [*] I have commented my code, particularly in hard-to-understand areas
- [*] I have made corresponding changes to the documentation
- [*] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

File: src/server/api/controllers/cvs.controller.js [hunk 1]
```diff
@@ -7,14 +7,10 @@ const moment = require('moment-timezone');
 const getCvs = async (title, limit) => {
   try {
     if (title) {
-      return await knex('cvs')
-        .select('*')
-        .where('title', 'like', `%${title}%`);
+      return await knex('cvs').select('*').where('title', 'like', `%${title}%`);
```
Comment (pdsorensen) [inline src/server/api/controllers/cvs.controller.js:10]: You shouldn't need to change this line.

File: src/server/api/controllers/cvs.controller.js [hunk 2]
```diff
@@ -7,14 +7,10 @@ const moment = require('moment-timezone');
...
     }
     if (limit) {
-      return await knex('cvs')
-        .select('*')
-        .limit({ limit });
+      return await knex('cvs').select('*').limit({ limit });
```
Comment (pdsorensen) [inline src/server/api/controllers/cvs.controller.js:13]: You shouldn't need to change this line.

File: src/server/api/controllers/cvs.controller.js [hunk 3]
```diff
@@ -27,9 +23,7 @@ const getCvs = async (title, limit) => {
 
 const getCvById = async (id) => {
   try {
-    const cvs = await knex('cvs')
-      .select('cvs.id as id', 'title')
-      .where({ id });
+    const cvs = await knex('cvs').select('cvs.id as id', 'title').where({ id });
```
Comment (pdsorensen) [inline src/server/api/controllers/cvs.controller.js:26]: You shouldn't need to change this line.

File: src/server/api/controllers/cvs.controller.js [hunk 4]
```diff
@@ -48,10 +42,8 @@ const editCv = async (cvId, updatedCv) => {
     });
 };
 
-const deleteCv = async (cvId) => {
-  return knex('cv')
-    .where({ id: cvId })
-    .del();
+const deleteCv = async (cvsId) => {
```
Comment (pdsorensen) [inline src/server/api/controllers/cvs.controller.js:45]: You shouldn't need to change this line.

File: src/server/api/routes/cvs.router.js [hunk 5]
```diff
@@ -92,10 +92,7 @@ router.post('/', (req, res) => {
     .catch((error) => {
       console.log(error);
 
-      res
-        .status(400)
-        .send('Bad request')
-        .end();
+      res.status(400).send('Bad request').end();
```
Comment (pdsorensen) [inline src/server/api/routes/cvs.router.js:95]: You shouldn't need to change this line.

**Review by pdsorensen — CHANGES_REQUESTED** [review]
You need to be able to change files without formatting them. Otherwise it's hard to detect relevant changes.

File: src/server/api/routes/cvs.router.js (hunk 5 above)
Comment (CharlesKimMaina) [reply src/server/api/routes/cvs.router.js:95]: okey,.i have redone it,.,.the newest code is attached below,., Thanks Patrick

**Pull request closed without merging**